test/functional/legacy/059_utf8_spell_checking_spec.lua

```diff
@@ -0,0 +1,953 @@
+-- Tests for spell checking with 'encoding' set to "utf-8".
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+local write_file = helpers.write_file
+
+describe("spell checking with 'encoding' set to utf-8", function()
+  setup(function()
+    clear()
+    execute("syntax off")
+    -- This file should be encoded in ISO8859-1.
```

Comment:
This might be over-engineering things, but perhaps the relevant text could be base64 encoded instead of having to manually ensure the encoding?